### PR TITLE
fix: stop sending HTTP requests to Kafka to see if it's alive

### PR DESCRIPTION
**Pull request by shawnsarwar**
The previous method of checking to see if Kafka was alive (sending a head request and having it reject it) was causing a warning in Kafka. As @tremolo discovered, the first 4 bytes of the Kafka protocol are for message size and it looked like we wanted to send a 1.2GB message to it.
```
# org.apache.kafka.common.network.InvalidReceiveException: Invalid receive (size = 1212498244 larger than 104857600)

In [8]: struct.unpack('!i', b'HEAD')
Out[8]: (1212498244,)
```
We've replaced that check with a simple socket.connect that works and doesn't offend Kafka.

### CODE REVIEW

File: aether-producer/producer/aether_producer.py [hunk 1]
```diff
@@ -165,21 +164,21 @@ def safe_sleep(self, dur):
 
     # Connectivity
 
+    # see if kafka's port is available
     def kafka_available(self):
-        kafka_url = "http://%s" % self.settings["kafka_url"]
+        kafka_ip, kafka_port = self.settings["kafka_url"].split(":")
+        kafka_port = int(kafka_port)
```
Comment (admbtlr) [inline aether-producer/producer/aether_producer.py:170]: Default to `80`/`443` in the (admittedly unlikely) event that there's no port?

Comment (shawnsarwar) [reply aether-producer/producer/aether_producer.py:170]: The Kafka port defaults to 9092. It shouldn't ever be on the default http/s ports

File: aether-producer/conf/pip/primary-requirements.txt [hunk 2]
```diff
@@ -17,6 +17,5 @@ flask
 gevent
 psycopg2-binary
 psycogreen
-requests
```
Comment (obdulia-losantos) [inline aether-producer/conf/pip/primary-requirements.txt:20]: If you change the primary requirements you should also change the frozen requirements.
`docker-compose -f docker-compose-connect.yml run --no-deps producer pip_freeze`

File: aether-producer/producer/aether_producer.py (hunk 1 above)
Comment (admbtlr) [reply aether-producer/producer/aether_producer.py:170]: OK, that's what I figured.

File: aether-producer/conf/pip/primary-requirements.txt (hunk 2 above)
Comment (shawnsarwar) [reply aether-producer/conf/pip/primary-requirements.txt:20]: I believe that `requests` is still required by `aether.client` so it's a dependency even though it's no longer a primary one.

**Review by admbtlr — APPROVED** [review]
Fine by me, although @obdulia-losantos 's comment still stands

**Pull request merged**